hazelcast/src/main/java/com/hazelcast/spi/impl/OperationScheduler.java

```diff
@@ -0,0 +1,49 @@
+package com.hazelcast.spi.impl;
+
+import com.hazelcast.nio.Packet;
+import com.hazelcast.spi.Operation;
+
+/**
+ * The OperationScheduler is responsible for scheduling operations to be executed locally.
+ */
+public interface OperationScheduler {
+
+    @Deprecated
```

Comment:
I don't see any point in @PrivateAPI on impl package; it is all internal so no need to say it is private.